packages/protocol/releaseData/initializationData/release4.json

```diff
@@ -1 +1,3 @@
-{}
\ No newline at end of file
+{
+  "GrandaMento": ["0x000000000000000000000000000000000000ce10", "0x0000000000000000000000000000000000000000", "0", "1209600"]
```

Comment:
this should actually go in `release5.json`
I know it's super confusing lol

Reply:
we should also have an empty `release4.json` for future reference, right?